packages/block-library/src/cover/style.scss

```diff
@@ -218,6 +218,6 @@ section.wp-block-cover-image > h2,
 	z-index: 1;
 	margin-bottom: 0;
 	max-width: $content-width;
-	padding: $block-padding;
+	padding: 0.45em;
```

Comment:
Technically, `0.4375em` would be closer to the original value. Even if you only go to 2 decimal places, it would be `0.44em`, not `0.45em`.

Reply:
But honestly, I'm not sure how much noticeable difference it will actually make either way, seeing as that these styles are targeting old deprecated versions of the Cover block, back when it was the Cover Image block.

Reply:
You are correct, `0.4375`would be the correct value. But in some places, I just tried to make it a bit saner. 0.45 _sounds_ better than `0.4375` or `0.44` and in this case it was inconsequential :relaxed:

Reply:
Changed to 0.44